config/samples/flows_v1beta1_flowcollector.yaml

```
@@ -49,6 +49,7 @@ spec:
     conversationTerminatingTimeout: 5s
     conversationHeartbeatInterval: 30s
     conversationEndTimeout: 10s
+    clusterName:
```

```suggestion
```

```suggestion
    # Append a unique cluster name to each record
    # clusterName: <CLUSTER NAME>
```

If the default is empty, then we can remove it from the sample or add a comment as example